### PR TITLE
mention the official docs and less canned sounding tone.

### PR DESCRIPTION
**Pull request by krasi-georgiev**
Signed-off-by: Krasi Georgiev <kgeorgie@redhat.com>

### CODE REVIEW

File: snippets/saved_replies.md [hunk 1]
```diff
@@ -24,9 +24,11 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
...
+To make your question, and all replies, easier to find, we suggest you move this over to our [user mailing list](https://groups.google.com/forum/#!forum/prometheus-users).
 
-Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+If you haven't looked already you might find the answer in the [official docs and examples](https://prometheus.io/docs/introduction/overview/) as well as searching in the google [users](https://groups.google.com/forum/#!forum/prometheus-users) or [devs](https://groups.google.com/forum/#!forum/prometheus-developers) groups or ask  in the #prometheus [IRC channel ](https://webchat.freenode.org/?channels=prometheus), and I am sure there will be someone to help you out.
```
Comment (brian-brazil) [inline snippets/saved_replies.md]: I don't see a need to mention that these are google groups - they're mailing lists. You've some extra spaces too.

Comment (brian-brazil) [inline snippets/saved_replies.md]: "and I am sure there will be someone to help you out." doesn't flow with the rest of the sentance

File: snippets/saved_replies.md [hunk 2]
```diff
@@ -24,9 +24,11 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
...
+
+Once your questions have been answered, please add a link to the solution to help other Prometheans in trouble reaching this from a search :+1:
```
Comment (brian-brazil) [inline snippets/saved_replies.md]: I don't think +1 is valid markdown

File: snippets/saved_replies.md (hunk 1 above)
Comment (RichiH) [reply snippets/saved_replies.md]: It might not be true, either. I try to avoid making promises.

File: snippets/saved_replies.md [hunk 3]
```diff
@@ -24,9 +24,12 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
...
+To make your question, and all replies, easier to find, I suggest you move this over to our `[user mailing list](https://groups.google.com/forum/#!forum/prometheus-users)`.
 
-Once your questions have been answered, please add a short line pointing to relevant replies in case anyone stumbles here via a search engine in the future.
+If you haven't looked already you might find your answer in the `[official docs and examples](https://prometheus.io/docs/introduction/overview/)` or by searching in the `[users](https://groups.google.com/forum/#!forum/prometheus-users)` or `[devs](https://groups.google.com/forum/#!forum/prometheus-developers)` groups.<br/>
```
Comment (brian-brazil) [inline snippets/saved_replies.md]: Why the backticks?

File: snippets/saved_replies.md [hunk 4]
```diff
@@ -24,9 +24,12 @@ It's not always feasible, or even possible, to fill out the XXX used in the snip
...
+The #prometheus `[IRC channel ](https://webchat.freenode.net/?channels=prometheus)` is also a great place to  mix up with the community and ask questions(don't be afraid to answers few while waiting) `:+1:`.
```
Comment (brian-brazil) [inline snippets/saved_replies.md]: I'd keep this focused on the user's question.

File: snippets/saved_replies.md (hunk 3 above)
Comment (krasi-georgiev) [reply snippets/saved_replies.md]: otherwise you loose the formatting when you copy and paste.

Comment (brian-brazil) [reply snippets/saved_replies.md]: Does the canned reply feature of github not handle this correctly?

Comment (krasi-georgiev) [reply snippets/saved_replies.md]: nope , you need to open the raw file to copy and paste the text with the code

File: snippets/saved_replies.md (hunk 4 above)
Comment (krasi-georgiev) [reply snippets/saved_replies.md]: @RichiH what do you think?

**Comment by krasi-georgiev** [comment]
feel free to ping me and I will reopen if you think this change would be useful.

**Pull request closed without merging**